docs-ui/stories/core/colors/tables.tsx

```diff
@@ -6,15 +6,9 @@ import space from 'sentry/styles/space';
 
 import ColorSwatch from './colorSwatch';
 
-type ColorDefinition = {
-  name: string;
-  lightValue: string;
-  darkValue: string;
-};
-
 type ColorGroup = {
   id: string;
-  colors: ColorDefinition[];
+  colors: string[];
```

Comment:
Should this be `Array<keyof typeof darkTheme | keyof typeof lightTheme>`?

Reply:
@vuluongj20 ^

Reply:
Good point! Just updated this